packages/desktop/webpack.config.js

```diff
@@ -124,6 +126,7 @@ const mainPlugins = [
         APP_NAME: JSON.stringify(appName),
         APP_ID: JSON.stringify(appId),
         'process.env.STAGE': JSON.stringify(stage),
+        'process.env.APP_PROTOCOL': JSON.stringify(appProtocol),
```

Comment:
Is it better to have it just a floating global constant or as `process.env.VARIABLE`?

Reply:
it doesn't show linting errors when we use process.env